upgrades/steps_263.go

```diff
@@ -23,6 +27,6 @@ func resetKVMMachineModificationStatusIdle(context Context) error {
 		// machines.
 		return nil
 	}
-	client := context.APIState().UpgradeSteps()
+	client := upgradesteps.NewClient(context.APIState())
```

Comment:
Fixes a problem where connectivity to the container machines are lost after upgrade to 2.6.3.  The agent is running but the containers loose their ip addresses.